.github/workflows/maven-checks.yml

```
@@ -14,17 +14,18 @@ env:
-  # that each commit to release-0.293-clp-connector is checked completely.
-  cancel-in-progress: "${{github.ref != 'refs/heads/release-0.293-clp-connector'}}"
+  # Cancel in-progress jobs for efficiency. Exclude branches with `release-0.293-clp-connector-snapshot` prefix so
+  # that each commit to these branches is checked completely.
+  cancel-in-progress: "${{!startsWith(github.ref, 'refs/heads/release-0.293-clp-connector-snapshot')}}"
 
 jobs:
   maven-checks:
     strategy:
       fail-fast: false
       matrix:
         java: [ 8.0.442, 17.0.13 ]
```

YAMLlint (1.37.1)

[error] too many spaces inside brackets

(brackets)